MobileWallet/Common/Extensions/UIView.swift

```diff
@@ -72,4 +72,31 @@ extension UIView {
         border.frame = CGRect(x: frame.size.width - borderWidth, y: 0, width: borderWidth, height: frame.size.height)
         addSubview(border)
     }
+
+    func applyGradient(animated: Bool = false) {
+        let gradient = CAGradientLayer()
+        gradient.frame = bounds
+        gradient.colors = [
+            Theme.shared.colors.gradient1!.cgColor,
```

Comment:
If it's just 2 colors then naming them `gradientStart` and `gradientEnd` may read better.